### PR TITLE
FT-62 #Allow image to be removed from model

### PR DESCRIPTION
**Pull request by simonmarklar**
- Added clearImage function
- Added tests
- Updated documentation
- Updated example
- general housekeeping

### CODE REVIEW

File: dist/bm-angular-camera.js [hunk 1]
```diff
@@ -130,12 +131,18 @@ function CameraController($scope, $element, Camera) {
     });
   };
 
+  vm.clearImage = function clearImage() {
+    if ($window.confirm(CLEAR_CONFIRM)) {
```
Comment (mymattcarroll) [inline dist/bm-angular-camera.js]: Would you be better to expose an optional `beforeClear()` function via a `&?` binding? Implementations might want to use a nice bootstrap or material model for confirmation.

Comment (simonmarklar) [inline dist/bm-angular-camera.js]: For the demo sam asked for a javascript popup confirm, but that would be a nice feature. 

i'll make an issue for it so i can get on with other issues that need to be done for the demo

Comment (simonmarklar) [inline dist/bm-angular-camera.js]: https://github.com/blinkmobile/angular-camera/issues/3

**Comment by mymattcarroll** [comment]
LGTM 👍

**Pull request merged**